src/components/notFoundPage.tsx

```diff
@@ -0,0 +1,9 @@
+export const NotFoundPage = () => {
```

Comment:
Consider changing the filename to NotFoundPage.tsx to match the name of the primary export 'NotFoundPage'.

Reply:
Same. Not fixed